src/Opdex.Platform.WebApi/Startup.cs

```diff
@@ -298,7 +309,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, Telemetr
         }
 
         var builder = configuration.DefaultTelemetrySink.TelemetryProcessorChainBuilder;
-        builder.UseAdaptiveSampling(maxTelemetryItemsPerSecond:5, excludedTypes:"Exception");
+        builder.UseAdaptiveSampling(maxTelemetryItemsPerSecond:5, excludedTypes:"Trace;Exception");
```

Comment:
This will prevent warning logs being sampled. Thought it'd be useful to try this out. Might have slightly degraded performance, although our avg request time is very low at the moment